--- freezegun-0.3.12/tests/test_ticking.py.orig	2019-12-28 20:04:27.061373016 +0100
+++ freezegun-0.3.12/tests/test_ticking.py	2019-12-28 20:20:15.159570062 +0100
@@ -1,6 +1,9 @@
 import datetime
 import time
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 from freezegun import freeze_time
--- freezegun-0.3.12/tests/test_utils.py.orig	2019-12-28 20:04:27.061373016 +0100
+++ freezegun-0.3.12/tests/test_utils.py	2019-12-28 20:28:53.326762910 +0100
@@ -1,6 +1,9 @@
 from unittest import SkipTest
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from freezegun import api
 from tests import utils
